script/build-icons

```diff
@@ -1,11 +1,19 @@
 #!/bin/sh
 
-dir=$(cd $(dirname $0) && cd "../src/common/icons" && pwd)
-src="${dir}/icon.svg"
+dir="$(cd "$(dirname "$0")" && cd "../src/common/icons" && pwd)"
+dark_src="${dir}/icon.svg"
+light_src="${dir}/icon-light.svg"
+
+export_png_icon() {
+  src=$1 dest=$2
+  inkscape -z -o "$dest" -D -w "$s" "$src" # export
+  convert "$dest" -background none -gravity center -extent "${s}x${s}" "$dest" # pad
+  pngcrush -brute -ow "$dest" # reduce
+}
 
 for s in 16 32 48 64 128; do
-  dest="${dir}/icon-${s}.png"
-  inkscape -z -e $dest -D -w $s $src # export
```

Comment:
FYI: the `-e` flag is apparently no longer supported (at least Inkscape errors for me when I try this...) 💥 

I could not find the proper documentation, but I think™ that `-o` should be the equivalent for current versions:
```
File export:
  -o, --export-file=EXPORT-FILENAME          Output file name (file type is guessed from extension)
  --export-overwrite                         Overwrite input file
  --export-type=[...]                        File type(s) to export: [svg,png,ps,eps,pdf,tex,emf,wmf,xaml]
```